docs/installation.rst

```diff
@@ -8,38 +8,37 @@ Installation
 
 .. code-block:: bash
 
-    $ git clone https://github.com/BeyondTheClouds/enos
-
-You should also choose to go with a virtualenv. Please refer to the `virtualenv
-<https://virtualenv.pypa.io/en/stable/>`_ documentation for further information.
+    $ pip install enos
```

Comment:
Yes, currently our master tracks kolla-ansible:stable/ocata, but I'd like it track kolla-ansible:master. 
After merging we could start a branch stable/ocata, tag it x.y.z and publish on pypi under the same version number.
Parallelly we can start working on supporting kolla-ansible:master from enos:master.